.github/workflows/deploy_sphinx_docs.yml

```diff
@@ -0,0 +1,60 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Sphinx Docs Deploy
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+
+on:
+  push:
+    branches:
+      - dev
+    paths:
+      - 'docs/**'
+      - '.github/workflows/deploy_sphinx_docs.yml'
+
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - uses: actions/checkout@v2
+
+    # Build Sphinx
+    - name: Sphinx Build
+      uses: ammaraskar/sphinx-action@0.4
+      env:
+        GH_TOKEN: ${{ secrets.GH_TOKEN }}
+      with:
+        # The folder containing your sphinx docs.
+        docs-folder: docs/
+        # The command used to build your documentation.
+        build-command: make html
+        # Run before the build command, you can use this to install system level dependencies, for example with "apt-get update -y && apt-get install -y perl"
+        #pre-build-command: # optional
+
+    - name: Commit documentation changes
+      run: |
+        git clone https://github.com/pkgpkr/Package-Picker.git --branch gh-pages --single-branch gh-pages
+        cd gh-pages
+        rm -r *
+        cp -r ../docs/build/html/* .
+        touch .nojekyll
+        git config --local user.email "action@github.com"
```

Comment:
Does it make sense to use our shared `pkgpkr1` account for this?

Reply:
@djsauble it is using that account's token to do that commit, but this was a clean way to specify that it's committed by GithubAction..  idk, I kind of like as is, but we could plug in another name and email if you'd like, I'm cool with either. but technically it's just metadata on the commit

Reply:
Naw, this is fine. I've seen some companies have a "bot" user for this purpose, maybe we spin up a `bot@pkgpkr.com` email eventually.

Reply:
Oh i see, sure, why not, maybe between 5/6 and 5/14 : )